README.md

```diff
@@ -18,47 +18,47 @@ To understand what exactly this keyboard is, please watch the video "https://www
 
 So, what capablities does 8Vim have?
 
-##### Basic Needed Stuff
+##### Basic Stuff
 
-- Right Sector acts as a backspace Key.
-- Bottom Sector acts as an Enter Key.
-- Top Sector acts as a combo of SHIFT and CAPS_LOCK Key, i.e, Press once Shift is active, Press Twice CAPS is active and Press once more and everything is back to usual.
-- Left Sector acts as the button that takes you to the Number Pad.
+- right sector acts as a Backspace key.
+- bottom sector acts as an Enter Key.
+- top sector acts as a combo of Shift and Caps Lock key (i.e., press once to activate Shift. Press again for Caps Lock. Press once more and everything is back to normal.)
+- Left Sector acts as the button that takes you to the number pad.
```

Comment:
If the bullet-points above aren't capitalized, i think the last one one shouldn't be either.

Reply:
Good catch! Fixed it